documentation/swagger-authoring-automated-guidelines.md

```diff
@@ -1,18 +1,22 @@
-# Automated Guidelines for Swagger Authoring #
+# Automated Guidelines for OpenAPI(swagger) specs Authoring #
 
-This document lists the set of automated rules that can be validated against swagger spec by running [validation tools](https://github.com/Azure/adx-documentation-pr/blob/master/tools/tools.md). Please look [here for Manual guidelines](swagger-authoring-manual-guidelines.md).
+This document lists the set of automated rules that can be validated against OpenAPI(swagger) spec by running [validation tools](https://github.com/Azure/adx-documentation-pr/wiki/OpenAPI-Validation-tools). Please look [here for Manual guidelines](swagger-authoring-manual-guidelines.md).
 
 It is a requirement to conform to all manual and automated rules with severity "Error" before sending a pull request for review.
 
+We request OpenAPI(Swagger) spec authoring be assigned to engineers who have an intimate knowledge of a service endpoint and its developer experience to avoid feeding inefficiencies into teams that focus on Azure developer experiences and the rest of the Azure eco-system.
+
 ## Error vs Warning
-- Rules with severity "Error" have to be addressed for the swagger to be approved by the reviewers. If there is a strong reason for why the rule cannot be addressed in a swagger spec, escalation process has to be followed to get sign off from management.
+- Rules with severity "Error" have to be addressed for the OpenAPI(swagger) spec to be approved by the reviewers. If there is a strong reason for why the rule cannot be addressed in an OpenAPI(swagger) spec, [suppression process](https://github.com/Azure/adx-documentation-pr/wiki/Swagger-Validation-Errors-Suppression) has to be followed to get sign off.
```

Comment:
not all "errors" are allowed to be suppressed, so we may need to state that here. "If there's a strong reason for why the rule cannot be addressed in a spec and it'll be a permanent exception, then the suppression process should be followed."